templates/.github/workflows/ci.yml

```diff
@@ -16,6 +16,9 @@ env:
   accountId: ${{ secrets.AWS_ACCOUNT_ID }}
   repo: ${{ secrets.ECR_REPOSITORY }}
   cluster-name: ${{ secrets.EKS_CLUSTER_NAME }}
+  # Note that technically these would change per environment, but there is only one environment for the Playground anyway
+  APP_NAME: <% .Name %>
```

Comment:
@davidcheung Do you have any preference here between `APP_NAME` vs `app-name`? I kept them as `APP_NAME` / `NAMESPACE` for consistency with the old environment variables and because they're not quite the same as the environment variables above (as per the commented note), but I don't have a strong preference.

Reply:
Yeah I don't have a strong preference either, lets keep them as-is? 
depends how theyre used, if i end up using `${{ env.app-name }}` with the github expression syntax they may look better that way, but if its in a command like `echo $APP_NAME` i tend to use the uppercase.